@kiva/kv-components/vue/KvChip.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<div class="tw-inline-flex">
+		<button
+			class="
+			tw-bg-secondary
+			hover:tw-bg-tertiary
+			tw-items-center
+			tw-flex
+			tw-flex-row
+			tw-cursor-pointer
+			tw-whitespace-nowrap
+			tw-rounded
+			tw-h-2
+			tw-py-2
+			tw-px-2
+			tw-mr-2
```

Comment:
Maybe we should not include these margin classes. This will allow the component that implements this component to handle that. For example, it would be easier to throw multiple chips in a flex layout if they don't have margin

Reply:
@eddieferrer Are you referring to only the margin classes or the padding classes as well?

Reply:
the margin classes